src/table/__tests__/resizable-columns.test.tsx

```diff
@@ -219,43 +224,34 @@ test('takes width as min width if it is less than 120px and min width is not set
   const { wrapper } = renderTable(<Table {...props} />);
   expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '100px' });
 
-  fireMousedown(wrapper.findColumnResizer(1)!);
-  fireMouseMove(100);
-  fireMouseup(100);
+  firePointerdown(wrapper.findColumnResizer(1)!);
+  firePointermove(100);
+  firePointerup(100);
   expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '100px' });
 });
 
 test('should follow along each mouse move event', () => {
   const { wrapper } = renderTable(<Table {...defaultProps} />);
   expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '150px' });
 
-  fireMousedown(wrapper.findColumnResizer(1)!);
-  fireMouseMove(200);
+  firePointerdown(wrapper.findColumnResizer(1)!);
+  firePointermove(200);
   expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '200px' });
-  fireMouseMove(250);
+  firePointermove(250);
   expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '250px' });
-  fireMouseMove(200);
+  firePointermove(200);
   expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '200px' });
-  fireMouseup(200);
-  expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '200px' });
-});
-
-test('should allow column resize with missing mousemove event', () => {
```

Comment:
do you have any ideas why this test existed? seems a strange one, but slightly worrying...